tests/test_report.py

```diff
@@ -54,6 +53,7 @@ def test_make_whitelist(check_report):
 bar  # unused function ({filename}:7)
 _.foobar  # unused attribute ({filename}:8)
 foobar  # unused variable ({filename}:9)
+# unreachable code after 'return' in file {filename} at line 11
```

Comment:
I think we should use the same format for unreachable code as in the other cases after all: `# unreachable code after 'return' ({filename}:11)`